cmd/oceantv/broadcast/youtube.go

```diff
@@ -353,6 +354,22 @@ func getBroadcastStatus(svc *youtube.LiveBroadcastsService, id string) (string,
 	return resp.Items[0].Status.LifeCycleStatus, nil
 }
 
+// GetBroadcastScheduledStart gets the status of the broadcast with the provided ID.
+func GetBroadcastScheduledStart(svc *youtube.Service, id string) (string, error) {
+	return getBroadcastStatus(youtube.NewLiveBroadcastsService(svc), id)
+}
+
+func getBroadcastScheduledStart(svc *youtube.LiveBroadcastsService, id string) (string, error) {
+	resp, err := svc.List([]string{"snippet"}).Id(id).Do()
+	if err != nil {
+		return "", fmt.Errorf("could not list broadcasts: %w", err)
+	}
+	if len(resp.Items) == 0 {
```

Comment:
Should we also check for more than one ? It's a really unexpected condition because you'd think that IDs are unique, but this code only works if this assumption holds. I would even panic under that condition.

Reply:
I think it's okay to proceed even if multiple are listed. We'll only use the first one right?

Reply:
Isn't that weird though ?

Reply:
I think we should at least log.

Reply:
Yeah that would be weird, a log sounds good so we can at least tell if there's weirdness going on.